tasks/request_files/test/unit_tests/test_request_files.py

```diff
@@ -966,20 +977,22 @@ def test_restore_object_client_error_raises(self, mock_logger_info: MagicMock):
             )
 
     # noinspection PyUnusedLocal
-    @patch("cumulus_logger.CumulusLogger.error")
     @patch("cumulus_logger.CumulusLogger.info")
-    def test_restore_object_client_error_last_attempt_logs_and_raises(
-        self, mock_logger_info: MagicMock, mock_logger_error: MagicMock
+    def test_restore_object_202_returned_raises(
+        self, mock_logger_info: MagicMock
     ):
+        """
+        A 202 indicates that the file is already restored, and thus cannot presently be restored again.
+        Should be raised.
+        """
         glacier_bucket = uuid.uuid4().__str__()
         key = uuid.uuid4().__str__()
         restore_expire_days = randint(0, 99)  # nosec
         retrieval_type = uuid.uuid4().__str__()
-        expected_error = ClientError({}, "")
         mock_s3_cli = Mock()
-        mock_s3_cli.restore_object.side_effect = expected_error
+        mock_s3_cli.restore_object.return_value = {"ResponseMetadata": {"HTTPStatusCode": 202}}
```

Comment:
The status value should be `200` if the granule has already been restored. `202` occurs if it is an initial successful restore.